addons/account/tests/test_account_payment_register.py

```diff
@@ -115,7 +148,14 @@ def setUpClass(cls, chart_template_ref=None):
             'partner_id': cls.partner_a.id,
             'invoice_line_ids': [(0, 0, {'product_id': cls.product_a.id, 'price_unit': 1600.0, 'tax_ids': []})],
         })
-        cls.in_refund_1.action_post()
+        cls.in_refund_2 = cls.env['account.move'].create({
+            'move_type': 'in_refund',
+            'date': fields.Date.today(),
+            'invoice_date': fields.Date.today(),
+            'partner_id': cls.partner_a.id,
```

Comment:
Rohh sorry, you also need the same partner.

Reply:
You are right I don't know how I missed that. Thought they were all partner A, will fix now. Thanks!